tests/functional/bounced_email.js

```diff
@@ -94,4 +101,46 @@ define([
     }
 
   });
+
+  const setUpBouncedSignIn = thenify(function () {
+    const client = new FxaClient(AUTH_SERVER_ROOT, {
+      xhr: nodeXMLHttpRequest.XMLHttpRequest
+    });
+
+    const email = TestHelpers.createEmail('sync{id}');
+
+    return this.parent
+      .then(clearBrowserState({ force: true }))
+      .then(createUser(email, PASSWORD, { preVerified: true }))
+      .then(openPage(`${config.fxaContentRoot}signin?context=fx_desktop_v2&service=sync`, '#fxa-signin-header'))
+      .then(respondToWebChannelMessage('fxaccounts:can_link_account', { ok: true }))
+      .then(fillOutSignIn(email, PASSWORD))
+      .then(testElementExists('#fxa-confirm-signin-header'))
```

Comment:
Can you replace all the inline selectors with entries in [selectors.js](https://github.com/mozilla/fxa-content-server/blob/master/tests/functional/lib/selectors.js)? If a selector/screen doesn't exist, feel free to add it. I'm trying to rip out inline selectors so that we can make updates in one place. I found that handy when making big changes in the email-first flow.